server/database/schemas/users.py

```diff
@@ -1,28 +1,44 @@
 from pydantic import BaseModel
 from uuid import UUID
+from datetime import datetime
 
 
 class UserBase(BaseModel):
     """Common user data shared by all schemas"""
 
-    email: str
-    username: str
+    email: str | None = None
+    username: str | None = None
 
 
 class UserGet(UserBase):
     """User data which can be read"""
 
     id: UUID
-    display_name: str
-
-    class Config:
-        orm_mode = True
+    display_name: str | None = None
 
 
 class UserCreate(UserBase):
     """User data which can be written"""
 
     password: str
+    display_name: str | None = None
 
     class Config:
         orm_mode = True
+
+
+class UserAuth(UserBase):
+    """User data for authentication"""
+
+    token: str
+    token_expiration_date: datetime
+
+    class Config:
+        orm_mode = True
+
+
+class UserLogin(BaseModel):
+    """User data for logging in"""
+
+    login: str
```

Comment:
It needs comments what could be placed here and what would be the behaviour of the server. It would be perfect, if in autogenered fast api docs would be an example of values

Reply:
Fixed